server/src/main/java/com/server/loan/calculator/validator/AdminDataValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.server.loan.calculator.validator;
+
+import com.server.loan.calculator.exception.ValidationException;
+import com.server.loan.calculator.model.AdminData;
+import io.micrometer.common.util.StringUtils;
+import lombok.RequiredArgsConstructor;
+
+import java.math.BigDecimal;
+
+@RequiredArgsConstructor
+public class AdminDataValidator {
+    private final AdminData adminData;
+    private final BigDecimalValueInRangeValidator valueInRangeValidator = new BigDecimalValueInRangeValidator();
+    private final BigDecimalInputAsCorrectNumberValidator inputValidator = new BigDecimalInputAsCorrectNumberValidator();
```

Comment:
with @ RequiredArgsConstructor annotation you dont need to instantiate these classes, if they are annotated with @ Component or @ Service annotations, which they should be